site/docs/v1/tech/guides/advanced-registration-forms.adoc

```diff
@@ -675,6 +675,12 @@ The form will fail validation unless the same value is entered in both fields.
 
 If [field]#Validation# is enabled, a regular expression must be specified. The user input will be matched against the regular expression and validation will fail if it doesn't match. See the https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html[Java Regular Expression documentation] for more information on how to build such a regular expression.
 
+=== Lambda Validation
+
+If you need to perform more complex validation of the registration form, including making `fetch` calls to an external
```

Comment:
Is this lambda a premium feature?

Reply:
Advanced registration forms are a premium feature and this only works on those

Reply:
Fair. But advanced registration forms are available with 'starter' plans, but lambda http connect is not.

So maybe change this to something like:

"If you need to perform more complex validation of the registration form, use this lambda. If you are on a plan which includes HTTP Lambda Connect, you can also make fetch calls to external APIs if needed to validate a registration."